stm32/unittest/src/tst_api_demod.c

```diff
@@ -93,20 +97,25 @@ void my_datatx(void *callback_state, unsigned char *packet, size_t *size) {
     *size = 0;
 }
 
+#define SPARE_RAM 10000
 
 int main(int argc, char *argv[]) {
+    char           dummy[SPARE_RAM];
     int            f_cfg, f_in, f_out;
```

Comment:
@donmr I added this dummy buffer in order to trigger an out of memory assert should the complete program have less that 10k free.  Any other ideas on how to do this?